docker-compose.yml

```
@@ -1,15 +1,16 @@
+      - "5432:5432"
     volumes:
-      - mysql_data:/var/lib/mysql
-volumes:
-  mysql_data:
\ No newline at end of file
+      - ./init:/docker-entrypoint-initdb.d
+
+  adminer:
+    image: adminer:latest
+    ports:
+      - "8080:8080"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)